data/human/news.txt

```diff
@@ -1086,6 +1190,37 @@ news "human syndicate manager"
 			"I would love to tell you about my job, but just telling you I work for the Syndicate probably already violates the non-disclosure agreement I signed."
 			"Those union boys are completely unreasonable; if we give them a finger then they try to eat up the whole arm up to the elbow."
 			"I've some disagreements with a manager above me. Can you pick up a mediator called Harry the Butcher from Deadman's Cove to help us resolve the problem peacefully?"
+			"I'm a manager, the folks here need to learn that it is my way or the highway."
+			"Those union guys wanted to rally on a planet nearby, but unionising is forbidden on that planet, so we could stop them easily by closing the spaceport there for known criminals!"
+			"My employees are angry when they need to do unpaid overtime in exceptional situations, they should have realized that working for me is exceptional every day."
+			"Patience is a virtue, but why is 'just hurry up' not also a virtue? Hurrying up works much better in the factory."
+			"When everything is under control, then we are not going fast enough. Except of course in your business, where people typically die due to explosions or due to the vacuum of space when things go out of control."
+			"I allow people to go home when they are no longer productive, even if it is 3 PM. Who are we kidding if we just steer on being present instead of steering on being productive?"
+			"We used to give our top performers shuttles, but they flew away faster than we could hire. Now we give them apartments and many are settling here and starting families."
+			"I'm really fed up with employees that have a 'can do' attitude. Just do your job, without the attitude."
+			"I'm actively recruiting people with a 9-to-5 mentality, because I know how important it is to have a good work/life balance."
+			"We used to advertise our food products with 100% organic ingredients, but nobody believed us. Now we are advertising with 60% organic ingredients and sales are sky high."
+			"We used to advertise that we are best in the marked, but that didn't work. Now we advertise with the slogan 'just what is needed to get the job done' and that really caught on."
+			"We hired a marketing agency and they came up with the sloppy slogan 'just what is needed to get the job done'. Can you believe we spent huge amounts of money on that garbage?"
+		word
+			`"`
+
+
+
+news "human work coach"
+	location
+		government "Syndicate"
+	name
+		word
+			"Corporate Consultant"
+			"Work/Life balance Coach"
+			"Work efficiency Coach"
+	message
+		word
+			`"`
+		word
+			"A startup company once hired me, and I immediately advised them to stop their company; Hiring an expensive consultant like me is a horrible decision for a cash strapped startup."
+			"Working as a high-performer requires sound decision making abilities. Hiring an expensive consultant like me usually suggests a lack thereof, but I'm not complaining since I get paid well."
```

Comment:
```suggestion
			"Working as the head of a company requires sound decision making abilities. Hiring an expensive consultant like me usually suggests a lack thereof, but I'm not complaining since I get paid well."
```